CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # CHANGELOG
 
+## 1.0.0-beta.7
+
+### Features
+
+- renames Error Icon to AlertCircle which is what the figma calls it
+- puts the old Warning triangle icon back
+- fixes the conditions in Alert component, to show the icon AlertCircle for the error and warning variants
```

Comment:
nit: I dunno if this counts as a "bug fix" or not

Reply:
it was not a bug, it was doing the right thing, just on the wrong icon? I think designs may have changed after the Alert was added in - I do remember thinking it is odd to use the 'i' icon for warning 😅